contracts/Tokens/Prime/Prime.sol

```diff
@@ -0,0 +1,563 @@
+pragma solidity 0.8.13;
+
+import "@openzeppelin/contracts-upgradeable/access/Ownable2StepUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC20/utils/SafeERC20Upgradeable.sol";
+import "@venusprotocol/governance-contracts/contracts/Governance/AccessControlledV8.sol";
+import "./PrimeStorage.sol";
+import "./libs/Scores.sol";
+
+interface IVToken {
+    function borrowRatePerBlock() external view returns (uint);
+
+    function reserveFactorMantissa() external returns (uint);
+
+    function totalBorrows() external returns (uint);
+
+    function accrueInterest() external returns (uint);
+
+    function borrowBalanceStored(address account) external returns (uint);
+
+    function exchangeRateStored() external returns (uint);
+
+    function balanceOf(address account) external returns (uint);
+
+    function underlying() external returns (address);
+}
+
+interface ERC20Interface {
+    function decimals() external view returns (uint8);
+}
+
+interface IXVSVault {
+    function getUserInfo(
+        address _rewardToken,
+        uint256 _pid,
+        address _user
+    ) external view returns (uint256 amount, uint256 rewardDebt, uint256 pendingWithdrawals);
+}
+
+interface IProtocolShareReserve {
+    enum Schema {
+        ONE,
+        TWO
+    }
```

Comment:
After having the npm package published (I would also import the interface from it), we should update this enum considering the last changes done [here](https://github.com/VenusProtocol/protocol-reserve/pull/2):
```
    enum Schema {
        DEFAULT,
        SPREAD_PRIME_CORE
    }
```